examples/multi_processor/README.md

```diff
@@ -0,0 +1,12 @@
+
+# Simple Trace Example
```

Comment:
Remove empty first line and change the title to indicate this is for multiple processors?

Reply:
Good point, have changed the title, and the description too.